package/libs/readline/Makefile

```diff
@@ -30,45 +29,39 @@ define Package/libreadline
   SECTION:=libs
   CATEGORY:=Libraries
   TITLE:=Command lines edition library
+  DEPENDS:=+libncursesw
   URL:=http://cnswww.cns.cwru.edu/php/chet/readline/rltop.html
   ABI_VERSION:=$(PKG_VERSION)
 endef
 
 define Package/libreadline/description
-	The Readline library provides a set of functions for use by applications 
-	that allow users to edit command lines as they are typed in. Both Emacs 
-	and vi editing modes are available. The Readline library includes 
-	additional functions to maintain a list of previously-entered command 
-	lines, to recall and perhaps reedit those lines, and perform csh-like 
+	The Readline library provides a set of functions for use by applications
+	that allow users to edit command lines as they are typed in. Both Emacs
+	and vi editing modes are available. The Readline library includes
+	additional functions to maintain a list of previously-entered command
+	lines, to recall and perhaps reedit those lines, and perform csh-like
 	history expansion on previous commands.
 endef
 
-# prevent "autoreconf" from removing "aclocal.m4"
-PKG_REMOVE_FILES:=
-
-CONFIGURE_ARGS += \
-	--enable-shared \
-	--enable-static \
-	--with-curses \
+CONFIGURE_ARGS += --with-curses --disable-install-examples
 
 CONFIGURE_VARS += \
 	bash_cv_wcwidth_broken=no \
 	bash_cv_func_sigsetjmp=yes \
 
-TARGET_CPPFLAGS:=-I. -I.. $(TARGET_CPPFLAGS)
-
 TARGET_CFLAGS += $(FPIC)
 
 define Build/InstallDev
 	$(INSTALL_DIR) $(1)/usr/include
 	$(CP) $(PKG_INSTALL_DIR)/usr/include/readline $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{history,readline}.{a,so,so.7,so.7.0} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{history,readline}.{a,so*} $(1)/usr/lib/
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/readline.pc $(1)/usr/lib/pkgconfig/
 endef
 
 define Package/libreadline/install
 	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{history,readline}.{so,so.7,so.7.0} $(1)/usr/lib/
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{history,readline}.so* $(1)/usr/lib/
```

Comment:
Please only bundle `*.so.8` and `*.so.8.0` but not the `*.so` symlink.

Reply:
Done